test/extensions/filters/udp/udp_proxy/hash_policy_impl_test.cc

```diff
@@ -50,6 +52,16 @@ class HashPolicyImplSourceIpTest : public HashPolicyImplBaseTest {
   const Network::Address::InstanceConstSharedPtr pipe_address_;
 };
 
+class HashPolicyImplKeyTest : public HashPolicyImplBaseTest {
+public:
+  HashPolicyImplKeyTest() : Key("key"), EmptyKey("") {}
+
+  void additionalSetup() override { hash_policy_config_->set_key(Key); }
+
+  const std::string Key;
+  const std::string EmptyKey;
```

Comment:
Better make the names conform the style: `key_` and `empty_key_`.

Reply:
Did it. Thanks.